server/api/classyear/classyear.controller.js

```diff
@@ -60,7 +74,10 @@ exports.create = function(req, res) {
           }
         }
       });
-      res.send(204);
+      ClassYear.getCurrent(function(err, currentClassYear){
+          global.currentClassYear = currentClassYear;
```

Comment:
Not right now, but I'd like it to be used everywhere (more efficient than asking the db constantly for the same data)